### DISCUSSION TITLE
Can you also see the camera feed?

### DISCUSSION THREAD

**Discussion by emilmort11** [humbertogontijo/homeassistant-roborock#186] [Q&A] [answered]
Was just wondering if this also allow you to get the camera feed from the onboard camera in home assistant and if not would this be posible to add in the future? Wanna clarify that i don't mean maps and stuff i mean the actual camera that you get when you control the vacuum

**Comment by jdejaegh** [comment, marked as answer]
Not currently, see #9  for discussion

**Reply by emilmort11** [reply]
ah thanks did not apparently look at all the open issue close enough! wish i was good enough to help out with making it work
